cosima_cookbook/querying.py

```diff
@@ -45,6 +68,22 @@ def get_ncfiles(session, experiment):
 
     return pd.DataFrame(q)
 
+def get_keywords(session, experiment=None):
+    """
+    Returns a set of all keywords, and optionally only for a given experiment
+    """
+
+    if experiment is not None:
+        q = (session
+            .query(NCExperiment)
+            .filter(NCExperiment.experiment == experiment))
+        r = q.one()
+        return r.keywords
```

Comment:
I think the following might work too:

```suggestion
        q = (session
            .query(NCExperiment.keywords)
            .filter(NCExperiment.experiment == experiment))
        return q.scalar()
```

Reply:
This worked:
```
return q.scalar().keywords
```
Tah.